src/engraving/libmscore/textbase.cpp

```diff
@@ -1545,7 +1545,6 @@ String TextBlock::text(int col1, int len, bool withFormat) const
 TextBase::TextBase(const ElementType& type, EngravingItem* parent, TextStyleType tid, ElementFlags f)
     : EngravingItem(type, parent, f | ElementFlag::MOVABLE)
 {
-    _textLineSpacing        = 1.0;
```

Comment:
This one should stay, as it is the one from TextBase... only the one from CharFormat should go.